policy/templates/el7/ci/goreleaser/goreleaser-el7.yml.d/builds.gotmpl

```diff
@@ -11,7 +11,9 @@ builds:
     goarch:
       - amd64
       - arm64
+{{ if has "s390x" .Branchvals.Features }}
```

Comment:
S/390 not officially supported on el7. If we ship this, folks will use it and if there is a problem, they will complain. I'm not sure we want to ship S/390 packages for 4-lts on el7 considering it will be EOL in June '24.

Reply:
This actually disables s390x for `release-4-lts`. We don't have `s390x` set as a feature for `release-4-lts` in the config. But since the el7 template didn't have the feature specific conditionals, it was always being rendered, even if the `s390x` feature isn't enabled. 
This fixes it, so that it's generated only if the s390x feature is set for el7 as well, the rendered result is here: https://github.com/TykTechnologies/tyk/pull/6227/commits/8df6506791351d178f9c6c9a3830e0107adb39c8